coral/src/app/layout/main-navigation/MainNavigation.tsx

```diff
@@ -39,36 +41,14 @@ function MainNavigation() {
             text={"Topics"}
           >
             <MainNavigationLink
-              href={"/coral/topics"}
               linkText={"All Topics"}
-              active={pathname === "/coral/topics"}
+              href={createNavLink(Routes.TOPICS)}
+              active={
+                pathname.startsWith(createNavLink(Routes.TOPICS)) ||
+                pathname.startsWith("/topic")
+              }
+              useRouter={true}
             />
-            {topicAclRequestEnabled ? (
```

Comment:
Goodbye, convenience link 🖖 The needs of the many outweigh the needs of the few.

Reply:
Sorryyyyyyyyyyyyyyyyyyyy 💐  I think it's a safer not to have it, as it's not clear why it's there.